vta/hardware/chisel/src/test/scala/StoreTests.scala

```diff
@@ -0,0 +1,103 @@
+// See LICENSE.txt for license details.
+package vta
+
+import chisel3._
+import chisel3.util._
+import chisel3.testers._
+import chisel3.iotesters.{PeekPokeTester, Driver, ChiselFlatSpec}
+
+class StoreTests(c: Store)(implicit val p: freechips.rocketchip.config.Parameters)
+    extends PeekPokeTester(c) {
+
+  val insn0 = "h00000008000800010000000000000029".U
+
+  val out_mem = List(
+    "hf6c4c7efc1ecf3fcfcfcfcfceec6e9f5".U, "hfcfcfbfcf3defcfafcfccdfcd3c6fcfc".U,
+    "hfcfcfcd8fcfcfcfcd1fcfcd0dbf8c7c7".U, "he2fcfcdddbfcfcfcfcd9c5f8fcf8ecfc".U,
+    "hf8e1fcf3fcfcdcd3fcfcfcfcfcfcfce3".U, "hfcfcfcfcfcfce9fcfafce9cafcfccefc".U,
+    "hd3fcfce7f2d2fcfcfcfcfcfcfcfcdef1".U, "hfafcf5fcfcc8d6fafcf1fcfcf6fcfcfc".U)
+
+  def run() {
```

Comment:
would you mind adding newlines to phrase the peek-expect blocks? Adding a few brief comments about e.g., why `step(2)` would also help make these tests more maintainable.